### PR TITLE
Add Vkontakte to the list of supported providers 

### PR DESCRIPTION
**Pull request by steveberdy**

### CODE REVIEW

File: src/OpenIddict.Client.WebIntegration/OpenIddictClientWebIntegrationProviders.xml [hunk 1]
```diff
@@ -1293,6 +1293,21 @@
...
+    <Environment Issuer="https://oauth.vk.com/">
+      <Configuration AuthorizationEndpoint="https://oauth.vk.com/authorize"
+      TokenEndpoint="https://oauth.vk.com/access_token"
+      UserinfoEndpoint="https://api.vk.com/method/users.get.json" />
+
+      <!--
+        Note: a scope is needed for authorization and can be bitmasked or listed as text
+      -->
+
+      <Scope Name="friends" Default="true" Required="false" />
```
Comment (kevinchalet) [inline src/OpenIddict.Client.WebIntegration/OpenIddictClientWebIntegrationProviders.xml:1307]: I see in the VK docs that they offer a `status` scope that seems more constrained. Would that work?

**Review by kevinchalet — COMMENTED** [review]
Thanks for your PR. As for the other providers: did you test it? 😃 

Also, please update the claims mapping handler.

File: src/OpenIddict.Client.WebIntegration/OpenIddictClientWebIntegrationProviders.xml (hunk 1 above)
Comment (steveberdy) [reply src/OpenIddict.Client.WebIntegration/OpenIddictClientWebIntegrationProviders.xml:1307]: I don't see why not. A scope is required, but the developers can decide if they don't want `status`.

**Comment by steveberdy** [comment]
I'm sorry, what do you mean by claims mapping handler? I'm guessing that's in the code? I haven't had a chance to look at this since I opened the PR.

File: src/OpenIddict.Client.WebIntegration/OpenIddictClientWebIntegrationProviders.xml (hunk 1 above)
Comment (kevinchalet) [reply src/OpenIddict.Client.WebIntegration/OpenIddictClientWebIntegrationProviders.xml:1307]: Please try replacing `friends` with `status`: if it works, let's use `status`.

**Comment by kevinchalet** [comment]
> I'm sorry, what do you mean by claims mapping handler? I'm guessing that's in the code?

Absolutely. I posted more information on one of your other PR: https://github.com/openiddict/openiddict-core/pull/1850#pullrequestreview-1557055346.

**Comment by kevinchalet** [comment]
Please let me know if you have any question or need help with your PRs 😃

**Comment by kevinchalet** [comment]
Closing due to inactivity.

**Pull request closed without merging**